.github/workflows/dotnet-core-win.yml

```diff
@@ -0,0 +1,48 @@
+name: .NET Windows
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-test:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
```

Comment:
need net46 and net461?

Reply:
We do since both are TFMs defined in the test projects.

I'm afraid this will be brittle since we'll end up changing TFMs in the test projects without adding them to the matrix.
A single test that ensures all tests in `test/**/*.csproj` are added here would cover for that. 
Would that be worth it?

Reply:
Hi @bruno-garcia , the idea here is to create parallel jobs instead a single on for windows. So, if we increase the number of tests for a specific framework, that wouldn't affect the others.

Reply:
Hi @cijothomas , yes, we need both because we have one project that targets net46 and one that targets net461

Reply:
It makes sense to me @eddynaka. 

I just mean it would be nice to make sure the TFMs stay in sync.
If we don't add some test to verify it, we'll need to ensure reviewers don't forget to point out that added/removed TFMs in `test/**/*.csproj` need to be reflected here

Reply:
Hi @bruno-garcia , can i ask what is TFM? Sorry for that.
About that, what i was thinking to do is: create a new CI where i will add the code coverage and test everything together.

Reply:
Sorry for that. TFM is short for target framework moniker. Like `net452` and `netcoreapp3.1`.
Right now you've added 4 of those which matches the current one used in the test projects. My concern is once we merge this, and change things (say we add `net5` or remove `net452`) to a test project, or on a new test project. 

Right now tests wouldn't run at all if we add a new TFM to a test project unless we remember to change the build matrix.

It's OK if we agree that we must make sure each time we change test projects TFMs we make sure there's a CI matrix target for that. But maybe it's worth the effort to add a single test (would run on all targets but that doesn't matter) that checks all `test/**/*.csproj <TargetFrameworks>` items and makes sure all of them exist in the GH actions config file.

Reply:
Hi @bruno-garcia , thanks for the explanation. My bad with the abbreviation.

Yeah, you are right! i will add a new ci with the merged stuff and one generic test. I won't add in the current one, because it would test in all options in the matrix that test.

Reply:
just pushed a new commit with the new CI. Let Me know what do you think! @bruno-garcia